makefiles/tools/openocd.inc.mk

```
@@ -38,6 +38,12 @@ ifneq (,$(OPENOCD_CMD_RESET_RUN))
   $(call target-export-variables,reset,OPENOCD_CMD_RESET_RUN)
 endif
 
+ifneq (,$(OPENOCD_CMD_RESET_HALT))
+  # Export OPENOCD_CMD_RESET_HALT only to the reset target
+  $(call target-export-variables,flash,OPENOCD_CMD_RESET_HALT)
+  $(call target-export-variables,flash-only,OPENOCD_CMD_RESET_HALT)
```

Does it work with the following:
```suggestion
  $(call target-export-variables,flash%,OPENOCD_CMD_RESET_HALT)
```